src/mergeSortLoop.js

```diff
@@ -0,0 +1,42 @@
+export default function mergeSort(toSort){
+  let currentLevel = []
```

Comment:
`.map` returns the collection created by applying the function specified as the parameter to each element in the original collection.  This could be leveraged to make lines 2-7 more concise:
```js
const currentLevel = toSort.map( number => [ number ] )
```